core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -23,6 +23,7 @@
 import mindustry.ui.*;
 
 import java.io.*;
+import java.net.URISyntaxException;
```

Comment:
why is this added?

Reply:
because of the catching URISyntaxException (i dont really know how it works)